runtime/src/consensus/registry.rs

```diff
@@ -64,7 +64,9 @@ pub struct TLSInfo {
     pub pub_key: signature::PublicKey,
 
     /// Public key that will be used for establishing connections after certificate rotation (if enabled).
-    pub next_pub_key: signature::PublicKey,
+    ///
+    /// Unused, certificates are not rotated anymore.
+    pub _deprecated_next_pub_key: Option<signature::PublicKey>,
```

Comment:
Add the `#[cbor(rename = "next_pub_key", optional)]` annotation.

Reply:
Can we remove these two fields, as runtimes will get upgraded after nodes?

Reply:
In theory runtimes could observe v2 descriptors in state and should be able to decode them?